client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -114,7 +114,7 @@ export const NonProductLineItem = styled( WPNonProductLineItem )< {
 	}
 `;
 
-const LineItem = styled( WPLineItem )< {
+export const LineItem = styled( WPLineItem )< {
 	theme?: Theme;
 } >`
```

Comment:
I'd extract the `{ theme?: Theme }` type as a named `ThemeProps` type into `theme.ts`. It's repeated quite often.